splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/SpliceSystemProcedures.java

```diff
@@ -1689,6 +1689,18 @@ static public void createSysUtilProcedures(List<Procedure> procedures) {
                 .ownerClass(UpgradeSystemProcedures.class.getCanonicalName())
                 .build();
         procedures.add(restartOlapServer);
+
+        procedures.add(Procedure.newBuilder().name("SYSCS_BACKUP_METADATA")
+                .numOutputParams(0).numResultSets(1).ownerClass(BackupSystemProcedures.class.getCanonicalName())
+                .varchar("directory", 32672)
```

Comment:
as I see the value 32672 is used in many places. I don't know, what does it mean exactly, but my guess based on the code, it's a limit of tables DDL. Maybe better to move it to the constant and documenting with javadoc. But clear, it's not related to this PR.

Reply:
Filed DB-11794